crates/oxc_semantic/src/symbol.rs

```diff
@@ -39,6 +39,10 @@ impl SymbolTable {
         self.spans.iter_enumerated().map(|(symbol_id, _)| symbol_id)
     }
 
+    pub fn iter_rev(&self) -> impl Iterator<Item = SymbolId> + '_ {
```

Comment:
I'm unsure whether this is a good public API. Are we able to directly call `iter().rev()` on call site?

Reply:
I initially thought the same, but the `iter` cannot call the `rev()`. Here is the  implementation of `iter`

https://github.com/oxc-project/oxc/blob/341081fbfacece1986e852acacfddfac806bfbd7/crates/oxc_semantic/src/symbol.rs#L38-L40